docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -52,7 +52,7 @@ Settings is a reference implementation. In a production environment, you should
 always run a transaction processor that supports the Settings transaction
 family.
 
-In this procedure, you will open six terminal windows on your host system: one
+In this procedure, you will open seven terminal windows on your host system: one
 for each Sawtooth component and one to use for client commands.
 
 .. note::
```

Comment:
How about adding a note about starting order for consensus engine?
It can be started before validator as well, unless otherwise explicitly specified by particular consensus algorithm.

Reply:
Thank you for the review.

Yes, the consensus engine can be started before, but then it doesn't connect and you don't see a connection message until after the validator started, making it harder to see problems.

Also, the consensus among Sawtooth developers is to keep this guide short and simple.

I added a note to the Sawtooth FAQ about this.